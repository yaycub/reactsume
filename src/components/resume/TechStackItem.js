import React from 'react';
import PropTypes from 'prop-types';

const TechStackItem = ({ item }) => {
  return (
    <p>
      {item}
    </p>
  );
};

TechStackItem.propTypes = {
  item: PropTypes.string.isRequired
};

export default TechStackItem;
