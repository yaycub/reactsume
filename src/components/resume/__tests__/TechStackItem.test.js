import React from 'react';
import { shallow } from 'enzyme';
import TechStackItem from '../TechStackItem';

describe('TechStackItem Component', () => {
  it('can render a TechStackItem', () => {
    const wrapper = shallow(<TechStackItem item={'im an item'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
