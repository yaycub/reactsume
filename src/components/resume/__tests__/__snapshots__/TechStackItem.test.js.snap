// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechStackItem Component can render a TechStackItem 1`] = `
<p>
  im an item
</p>
`;
